ssl/quic/quic_channel.c

```diff
@@ -1283,8 +1283,12 @@ static int ch_on_transport_params(const unsigned char *params,
                 goto malformed;
             }
 
-            /* Must match SCID of first Initial packet from server. */
-            if (!ossl_quic_conn_id_eq(&ch->init_scid, &cid)) {
+            /*
+             * Must match SCID of first Initial packet from server.
+             * Keep in mind the client always sends QUIC_TPARAM_INITIAL_SCID
+             * empty (cid.id_len == 0 (see ch_generate_transport_params()).
```

Comment:
(not reviewing - I don't know enough about quic - giving feedback on the comments) We seem to open two `(`s here, but only close one